test/fluri_test.dart

```diff
@@ -0,0 +1,107 @@
+library fluri.test.fluri_test;
+
+import 'package:fluri/fluri.dart';
+import 'package:test/test.dart';
+
+
+void commonFluriTests(dynamic getFluri()) {
```

Comment:
Purely curious - why does this take a function instead of the actual fluri object?